pomm-project/pomm-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "alias": "pomm",
```

Comment:
@Pierstoval the validation boot doesn’t failed, but the alias seems doesn’t work:
```
$ SYMFONY_ENDPOINT=https://symfony.sh/r/contrib/9 composer req pomm                                                         
                                                                                                                                                       
                                                                                                                                                       
  [UnexpectedValueException]    
  "pomm" is not a valid alias.  
                           
```